src/uuid.rs

```diff
@@ -44,7 +44,9 @@ impl IntoDart for Vec<uuid::Uuid> {
         use std::io::Write;
         let mut buffer = Vec::<u8>::with_capacity(self.len() * 16);
         for id in self {
-            let _ = buffer.write(id.as_bytes());
+            buffer
+                .write(id.as_bytes())
+                .expect("write uuids bytes to buffer");
```

Comment:
This will result in a panic during the runtime, is this something desirable? we can return a Result or impl it for Result only (not going to work we will need Specialisation in rust traits) But nonetheless, if this stays the case, we shall add a note in the docs that says the behaviour if this function will panic.

Reply:
Well earlier I just copy/pasted from the tests and saw it just now.

To be honest I didn't know about this so I looked [at the docs](https://doc.rust-lang.org/std/io/trait.Write.html#tymethod.write), and it seems like it's very unlikely to happen.

On top of it, the buffer is provisioned ahead with enough capacity to hold the incoming bytes.

> Each call to write may generate an I/O error indicating that the operation could not be completed. If an error is returned then no bytes in the buffer were written to this writer.

> It is not considered an error if the entire buffer could not be written to this writer.

Provided I understand the docs correctly, if I revert the code ignoring the error the side effect would be the bytes written to the buffer could be incomplete.

Reply:
I reverted and stated it in the docs 👌